### DISCUSSION TITLE
Realtime not showing all the visits/page views

### DISCUSSION THREAD

**Discussion by vitobotta** [plausible/analytics#1230] [Self-Hosted Support]
Hi! I noticed that the realtime page is not reporting everything for some reason. I know that Plausible is collecting all page views because if I go through pages of my website and I refresh the normal dashboard, I see the stats increasing (I'm testing with a website that doesn't have frequent visits yet so I know that it's my visits now).

But yeah, the realtime page is not reporting everything. Actually, it's not reporting anything for this website I am testing with, but it's reporting something for another website that has more visits. What can cause this? Thanks!

**Comment by ukutaht** [comment]
Is the `Pageviews (last 30 minutes)` number increasing when you refresh? It should.

Everything else like sources, pages, etc are deduplicated by visitors. So if you click around from the same IP address and user agent, you won't see any difference in sources/countries etc.

**Reply by vitobotta** [reply]
Yes the pageviews was increasing I think (I have temporarily uninstalled it and will reinstall because I'm still having problems with backups of Clickhouse).

I understand the deduplication, but some visits were not reported when I tried with NordVPN in different countries. It's not a problem with ad blockers because I am using a Cloudflare worker to proxy requests. I will try again once I reinstall. Thanks!
